test/wappsto_test.py

```diff
@@ -151,7 +151,12 @@ def setup_method(self):
                                                      (100, 100),
                                                      (0, 0),
                                                      (-1, None),
-                                                     (120, None)])
+                                                     (120, None),
+                                                     (-0.1, None),
+                                                     (0.1, 0),
+                                                     (3.3, 3),
+                                                     (3.0, 3),
+                                                     (3.9, 3)])
```

Comment:
Please add a test where the step size is something like "0.02" and the input data like "1.01 and 2.002"